lab-bundle-spec.md

```diff
@@ -158,21 +153,34 @@ environment_resources:
 
 ##### GCP Project (gcp_project)
 
-attribute                     | required | type    | notes
------------------------------ | -------- | ------- | --------------------------------------
-dm_template.script            |          | path    | Relative path to a Deployment Manager directory tree.
-dm_template.custom_properties |          | array   | Array of key/value pairs.
-fleet                         |          | enum*   | Specify a Qwiklabs fleet to pull the project from.
+attribute                           | required | type    | notes
+----------------------------------- | -------- | ------- | --------------------------------------
+dm_template.start.script            |          | path    | Relative path to a Deployment Manager directory tree.
+dm_template.start.custom_properties |          | array   | Array of key/value pairs.
+dm_template.end.script              |          | path    | Relative path to a Deployment Manager directory tree.
+dm_template.end.custom_properties   |          | array   | Array of key/value pairs.
+fleet                               |          | enum*   | Specify a Qwiklabs fleet to pull the project from.
+service_account_permissions         |          | array   | Array of project/roles(array) pairs to be granted to the default compute engine service account in this project.
```

Comment:
`service_account_permissions` was the best name I could come up with. Open to other suggestions.

Reply:
I'm mildly concerned about the ambiguity in the name, since there's a bunch of service accounts involved in the lab runtime. It's more verbose, but how about...

- `gce_service_account_permissions`,
- `default_service_account_permissions`, or
- `default_gce_service_account_permissions`?